rtrlib/rtr/packets.c

```diff
@@ -369,6 +369,11 @@ static bool rtr_pdu_check_size (const struct pdu_header *pdu) {
     //TODO: Check the error msg 0 termination
 
     break;
+  case SERIAL_QUERY:
+  case RESET_QUERY:
+  case RESERVED:
+  default:
```

Comment:
The PDUs are not checked because we will never receive them.
The serial query and reset query PDU are only send by the router.